examples/04_pong/main.rs

```diff
@@ -68,8 +68,8 @@ impl Plank {
     }
 }
 
-impl Component for Plank {
-    type Storage = VecStorage<Plank>;
+impl Component for Paddle {
+    type Storage = VecStorage<Paddle>;
```

Comment:
While we're at it, maybe we could use another storage?

Reply:
Like `DenseVecStorage`?

Reply:
Yes

Reply:
I don't think it will make any difference. Because there are just 2 paddles with `VecStorage` there will be just a `Vec` of length 2, which is ok.

Maybe merge this, and if I missed something @torkleyy will create another PR changing the storage?

Reply:
It doesn't make a difference in this particukar case. But we should not use `VecStorage` in our example, because I already see people using it as the default even though it can be very heavy if used incorrectly.

Reply:
I might have only used VecStorage until now? :thinking: